File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2441.py
k = [int(i) for i in input().split()]

a = k[0]
b = k[1]
c = k[2]

if a > b:
    temp = a
    a = b
    b = temp
    
if b > c:
    temp = b
    b = c
    c = temp
    
if a > b:
    temp = a
    a = b
    b = temp
    
tot = 0

fima = a+200
fimb = b+200
fimc = c+200

if fima < b:
    tot += b - fima

if fimb < c:
    tot += c - fimb
    
tot += a + 600 - fimc

print(tot*100)

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2441.c
#include <stdio.h>

int main(){
    int a, b, c, temp, tot, fima, fimb, fimc;
    scanf("%d %d %d", &a, &b, &c);
    
    if(a > b){
        temp = a;
        a = b;
        b = temp;
    }
    
    if(b > c){
        temp = b;
        b = c;
        c = temp;
    }
    
    if(a > b){
        temp = a;
        a = b;
        b = temp;
    }
    
    tot = 0;

    fima = a+200;
    fimb = b+200;
    fimc = c+200;
    
    if(fima < b){
        tot += b - fima;
    }
    
    if(fimb < c){
        tot += c - fimb;
    }
    tot += a + 600 - fimc;
    
    printf("%d\n", tot*100);
    
    return 0;
}